### ISSUE TITLE
Reimagining auto save / load

### ISSUE THREAD

**Issue by pajarorrojo** [batocera-linux/batocera-emulationstation#807]
Hello community

The auto save /load is a great new addon, but there are more and more options that hang from the auto save / load option im game menu. I think it is getting confusing and the new options do not make sense with the main option (auto save / load), I propose a reimagining of the concept with a "snapshot manager "option.

The current options are:

Auto save / load
   ON - use the autosave slot to automatically load games
   OFF - auto save / load off
   SHOW SAVE SNAPSHOTS -> shows the manager, manual choice
   SHOW IF SNAPSHOTS IS NOT EMPTY -> shows the manager only when there are previously saved games

I propose something simple and intuitive (in my opinion) such as:

SNAPSHOTS MANAGER
   OFF -> classic behavior, just start the game
   ON -> always shows the manager before starting a game (manual choice)
   AUTO SAVE / LOAD -> always use the autosave slot
   SHOW IF SNAPSHOTS EXIST -> show manager only if there are previously saved games.
   USE (L3) - pseudo off, only show whith (L3) pressing 

What do you think about?

Thanks again for the great work and community, batocera rules!!

In my opinion I think it is more logical and simple, it is also compatible with the future options of this great addon

**Comment by fabricecaruso** [comment]
What's the difference with the current options ? Except the names of the labels & the L3 thing. But the L3 button is already working in every cases (so it's not really a change).

**Comment by pajarorrojo** [comment]
Yes, you are right @fabricecaruso, there are no significant changes, basically just the labels. What I am trying to explain is to change the meaning, initially we only had two options: AUTO SAVE / LOAD = ON or OFF, ok it makes sense, but with the new additions, for example. "SHOW SNAPSHOTS IF NOT EMPTY" I think is confusing with the AUTO SAVE / LOAD? Question. My proposal is simply to change the concept so that all the options make sense with the main tag, a concept like "SNAPSHOT MANAGING" would make more sense to any option that hangs from it.
